README.md

```diff
@@ -1,2 +1,3 @@
 # mpgd-2024-pr
 hello, world
+# Krawuzikrabuzi
```

Comment:
I think this should be second level heading ("##" instead of "#")

Reply:
SICHA NED DU HALUNKE